README.md

```diff
@@ -1,35 +1,40 @@
-# FMPDO
+# FmPdo
 
-FMPDO is a php class that is designed to replace FileMaker.php when converting legacy FileMaker web applications to use a SQL database. The tables in the legacy solution can be converted to SQL all at once, or individually, as required.
+FmPdo is a drop-in php module for refactoring legacy FileMaker.php web applications to use a SQL database.
+ 
+FmPdo provides alternative implementation of FileMaker.php functions so that changes to existing application logic are
+reduced to the absolute minimum. In many cases, all that is required is refactoring the adapter configs.
 
+The legacy solution can be refactored to a SQL backend all at once, or incrementally, as required.
 
 ## Features
+
 ### Minimal changes to legacy apps
-* FMPDO methods calls and responses are the same format as FileMaker.php
+* FmPdo methods calls and responses are the same format as FileMaker.php
 
 ### Easy to Integrate
-* Convert all or some of your tables to SQL
+* Refactor all or some of your persistence to SQL tables
 
 ### Flexible
-* Choose any major SQL database, easily change databases at a later date (thanks to PDO)
-
-
+* Choose any major SQL database supported by PDO; easily change databases at a later date (thanks to PDO).
  
 ## System Requirements
 
-### PHP 5.3
-### PDO Driver for desired database
-* MySQL and SQLite are already in most PHP builds
-
-
+*  PHP 5.3 to 7.0
+    * PHP 5.3 and 5.4 support is deprecated
+    * PHP 5.5+ is recommended
+    * Use of (class name scalars)[https://wiki.php.net/rfc/class_name_scalars] is planned, and this will move minimum
+    version up to 5.5
+*  PDO Driver for desired database
+    * MySQL and SQLite are included by default in most PHP stacks
 
 
 ## License
 
-FMPDO is free for commercial and non-commercial use, licensed under the business-friendly standard MIT license.
+FmPdo is free for commercial and non-commercial use, licensed under the business-friendly standard MIT license.
 
 
-# FMPDO Documentation
+# FmPdo Documentation
 
 * index.php contains sample calls to supported methods, more formal docs are on the way
```

Comment:
I didn't remove the reference to `index.php`, but this whole section of the README should maybe be deleted until the content can be created.